vendor/bat-native-ads/src/bat/ads/internal/notifications.h

```diff
@@ -29,8 +29,9 @@ class Notifications {
   bool Get(const std::string& id, NotificationInfo* info) const;
 
   void Add(const NotificationInfo& info);
+  void Remove(); //removes the front item
```

Comment:
Can we please rename function to RemoveHead. We can then also remove the comment as the function name will help with readability